packages/terra-button-group/src/terra-dev-site/test/button-group/ProgrammaticDisabled.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import ButtonGroup from '../../../ButtonGroup';
+
+const ProgrammaticDisabled = () => {
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  return (
+    <ButtonGroup id="programmaticDisabledButtonGroup">
```

Comment:
Instead of having a similar example as of terra-button. we could have the button group example here. somewhat similar to the issue reported scenario.